### ISSUE TITLE
When I run deepstram-6.0 using my own generated .wts file and .cfg file, it keeps getting stuck

### ISSUE THREAD

**Issue by qybing** [marcoslucianops/DeepStream-Yolo#236]
2022-08-20T22:00:43.402918+0800 INFO | 1 | monitor:stdout_record[257] :[stdout] yolo_128

2022-08-20T22:00:43.403349+0800 INFO | 1 | monitor:stdout_record[257] :[stdout] Total number of YOLO layers: 367

2022-08-20T22:00:43.403774+0800 INFO | 1 | monitor:stdout_record[257] :[stdout] Building YOLO network complete

2022-08-20T22:00:43.404205+0800 INFO | 1 | monitor:stdout_record[257] :[stdout] Building the TensorRT Engine

2022-08-20T22:00:43.404623+0800 INFO | 1 | monitor:stdout_record[257] :[stdout] NOTE: letter_box is set in cfg file, make sure to set maintain-aspect-ratio=1 in config_infer file to get better accuracy

**Comment by qybing** [comment]
I am using yolo5

**Comment by qybing** [comment]
I am using yolov5

**Comment by marcoslucianops** [comment]
The process to create the engine may take long time.

**Comment by qybing** [comment]
> The process to create the engine may take long time.

No, it keeps looping and stuck at this line of code

**Comment by marcoslucianops** [comment]
Is there an error message (segmentation fault, etc)?

**Comment by qybing** [comment]
> Is there an error message (segmentation fault, etc)?

No, he didn't get any error, just stuck here in an infinite loop

**Comment by marcoslucianops** [comment]
Which board are you using?

**Comment by animeesh** [comment]
https://colab.research.google.com/drive/17HIHsKAdfTbZQSu7LxZIwtHQr7LZYSKD?usp=sharing this colab file can help you guys in .pt to .wts model conversion
